packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +63,48 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS || nothing}>
                     <slot></slot>
                 </div>
             </a>`;
     }
 
+    /**
+     * Generates padding for the component based on `padding` values passed
+     * by the consumer.
+     *
+     *
+     * Example: 'a' or 'a, b'
+     * Single values i.e `'a'` applies to all sides and `'a, b'` applies to: top, right, bottom, left
+     *
+     * @private
+     */
+    private generatePaddingCSS (): string {
+        const { padding } = this;
+        let paddingCSS = '';
+
+        // Check if padding is empty (null, undefined, or an empty string)
+        if (!padding) {
+            return '';
+        }
+
+        const paddingArray = padding
+            .split(',')
+            .map((item) => item.trim())
+            .filter((value) => value !== '');
+
+        if (paddingArray.length > 0) {
+            paddingCSS += `var(--dt-spacing-${paddingArray[0]})`;
```

Comment:
Do we want to validate that the values create valid tokens? Thinking in theory someone could try and pass something like `a); background-image: url(\"https://malicious.com/image.png\");` which would get through the processing we do above.

Reply:
Hard to exploit in practice, I know, but I think we know that all of the valid values for this array are the letters `a-j`, so we could validate that with a regex, perhaps

Reply:
![2023-09-27 16_48_34](https://github.com/justeattakeaway/pie/assets/26894168/ceb7e834-59d1-4107-9b09-7033186f188c)

Reply:
Should be fixed now.